.cargo/config

```diff
@@ -0,0 +1,4 @@
+[target.wasm32-wasi]
+rustflags = ["--cfg", "tokio_unstable"]
+#runner = ["enarx", "run", "--wasmcfgfile", "Enarx.toml"]
```

Comment:
Aren't we supposed to run this in Enarx? Why is this necessary, does this not work in Enarx? If so, what's the purpose of this PR?

Reply:
It works in Enarx, but Wasmtime is:
* Faster, and
* Presumably easier to get loaded in a CI environment.

Reply:
As per our conversation, I've removed the reference to Wasmtime, and changed the CI config to download and install Enarx, and confirmed that this works.